lib/utils/docker.coffee

```diff
@@ -137,15 +143,32 @@ exports.runBuild = (params, options, getBundleInfo, logStreams) ->
 	resolver = require('resin-bundle-resolve')
 	es = require('event-stream')
 	doodles = require('resin-doodles')
+	transpose = require('docker-qemu-transpose')
+	path = require('path')
 
 	logging = require('../utils/logging')
 
 	# The default build context is the current directory
 	params.source ?= '.'
 	logs = ''
+	# Only used in emulated builds
+	qemuPath = ''
 
-	# Tar up the directory, ready for the build stream
-	tarDirectory(params.source)
+	Promise.try ->
+		if options.emulated and platformNeedsQemu()
```

Comment:
can be `return if not (options.emulated and platformNeedsQemu())` and the rest of the method dedented